packages/server/modules/workspaces/services/projects.ts

```diff
@@ -169,22 +166,26 @@ export const moveProjectToWorkspaceFactory =
     }
 
     // Update roles for current project members
-    const projectTeam = await getProjectCollaborators({ projectId })
-    const workspaceTeam = await getWorkspaceRolesAndSeats({ workspaceId })
-    const {
-      defaultProjectRole: getDefaultProjectRole,
-      allowedProjectRoles: getAllowedProjectRoles
-    } = await getWorkspaceRolesAllowedProjectRoles({
-      workspaceId
-    })
+    const [workspace, projectTeam, workspaceTeam] = await Promise.all([
```

Comment:
while this could've been a duplicated version of the old function just for the new plan, the function was already adjusted to support both plans in the previous PR, so splitting it out into old & new would be redundant at this point (and removing the isNewPlan check should be easy)